test/test-framework/deploy/olm-catalog/memcached-operator/0.0.2/memcached-operator.v0.0.2.clusterserviceversion.yaml

```diff
@@ -4,96 +4,133 @@
+                - name: OPERATOR_NAME
+                  value: "memcached-operator"
   customresourcedefinitions:
     owned:
-      - description: Represents a cluster of Memcached apps
-        displayName: Memcached App
-        kind: Memcached
-        name: memcacheds.cache.example.com
-        version: v1alpha1
-        resources:
-          - kind: Deployment
-            version: v1
-          - kind: ReplicaSet
-            version: v1beta2
-          - kind: Pod
-            version: v1
-        specDescriptors:
-          - description: The desired number of member Pods for the deployment.
-            displayName: Size
-            path: size
-            x-descriptors:
-              - 'urn:alm:descriptor:com.tectonic.ui:podCount'
-        statusDescriptors:
-          - description: The current status of the application.
-            displayName: Status
-            path: phase
-            x-descriptors:
-              - 'urn:alm:descriptor:io.kubernetes.phase'
-          - description: Explanation for the current status of the application.
-            displayName: Status Details
-            path: reason
-            x-descriptors:
-              - 'urn:alm:descriptor:io.kubernetes.phase:reason'
+    - description: Represents a cluster of Memcached apps
+      displayName: Memcached App
+      kind: Memcached
+      name: memcacheds.cache.example.com
+      version: v1alpha1
+      resources:
+      - kind: Deployment
+        version: v1
+      - kind: ReplicaSet
+        version: v1beta2
+      - kind: Pod
+        version: v1
+      specDescriptors:
+      - description: The desired number of member Pods for the deployment.
+        displayName: Size
+        path: size
+        x-descriptors:
+        - 'urn:alm:descriptor:com.tectonic.ui:podCount'
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - 'urn:alm:descriptor:io.kubernetes.phase'
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - 'urn:alm:descriptor:io.kubernetes.phase:reason'
   keywords:
-    - memcached
-    - app
+  - memcached
+  - app
   displayName: Memcached Application
   provider:
     name: Example
     url: www.example.com
-  maturity: Basic Install
+  maturity: alpha
   version: 0.0.2
   replaces: memcachedoperator.v0.0.1
   maintainers:
-    - email: corp@example.com
-      name: Some Corp
+  - email: corp@example.com
+    name: Some Corp
   description: Main enterprise application providing business critical features with high availability and no manual intervention.
   labels:
-    alm-owner-enterprise-app: memcached-app-operator
-    alm-status-descriptors: memcached-app-operator.v0.0.2
+    alm-owner-enterprise-app: memcached-operator
+    alm-status-descriptors: memcached-operator.v0.0.2
```

Comment:
Do we need the above two labels?
I don't think they're adding any functionality to the CSV. Seem to be an optional label from previous iterations. And we're not selecting on these labels.
https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md#operator-metadata

Don't see these being used by any of the community operators as well.
https://github.com/operator-framework/community-operators/blob/master/community-operators/jaeger/jaeger.v1.8.2.clusterserviceversion.yaml#L75-L79

So probably remove these unless we have a reason to keep them in.